### PR TITLE
Removing preview dependencies

### PR DESCRIPTION
**Pull request by nemakam**

### CODE REVIEW

File: src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj [hunk 1]
```diff
@@ -31,10 +31,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Amqp" Version="[2.2.0, 3.0.0)" />
-    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.1.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="[1.0.1, 2.0.0)" />
```
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj]: Going from 1.1.0-preview to 1.0.1.
This is a downgrade, meaning features from 1.1.0-preview are not uesd?

Comment (SeanFeldman) [reply src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj]: `1.1.0-preview` was first and `1.0.1` was introduced _after_ (https://github.com/Azure/azure-sdk-for-net/issues/4121).
Asking as I'm using this for MSI and there are no release notes or PR/issue to determine what has changed in 1.1.0-prevew. Perhaps you folks know.

File: src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <Description>This is the next generation Azure Service Bus .NET Standard client library that focuses on queues &amp; topics. For more information about Service Bus, see https://azure.microsoft.com/en-us/services/service-bus/</Description>
-    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionPrefix>3.0.1</VersionPrefix>
```
Comment (SeanFeldman) [inline src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj]: Is this needed? `Verstion` is already set.

Comment (nemakam) [reply src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj]: Honestly, the whole thing is quite confusing. I'm trying to automate few systems. Some work with `<VersionPrefix>`, and some work with `<Version>`.. So I'm adding both of them.
 Do you see any concerns?

Comment (nemakam) [reply src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj]: I'll try to remove the `Prefix` version and see if I can change the automation to work without it.

**Pull request merged**